onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -249,6 +249,15 @@ def parse_arguments(argv: Optional[List[str]] = None) -> argparse.Namespace:
     )
     model_group.set_defaults(past_present_share_buffer=False)
 
+    model_group.add_argument(
+        "--use_decoder_masked_multihead_attention",
+        required=False,
+        action="store_true",
+        help="Uses `DecoderMaskedMultiheadAttention` to optimize the unidirectional decoding Attention computation. "
```

Comment:
nit: add description that the head size shall be 64 or 128

Since we did not verify this so add this could help user decide whether they can use this option.

Reply:
Done